plugins/woocommerce-blocks/tests/e2e/tests/checkout/checkout-block.merchant.block_theme.spec.ts

```diff
@@ -78,7 +78,7 @@ test.describe( 'Merchant → Checkout', () => {
 	} );
 
 	test.describe( 'Can adjust T&S and Privacy Policy options', () => {
-		test.beforeAll( async ( { browser } ) => {
+		test.beforeEach( async ( { browser } ) => {
 			const page = await browser.newPage();
 			await page.goto(
 				`${ process.env.WORDPRESS_BASE_URL }/?setup_terms_and_privacy`
```

Comment:
Nice catch, I missed this one! I'll move it to the folder where the other plugins are.

Reply:
OTOH, this seems to be a bit different case from the plugins in the e2e/plugins folder. They're installed but inactive until requested on a per-test basis. I'm not saying the woo-test-helper should not be there; it seems like it needs to be active by default, which is a bit different from the use case. Let's leave it for a follow-up PR, since this one's already pretty big! 🙈

Reply:
I just created https://github.com/woocommerce/woocommerce/issues/46745, to take care of relocating the helper plugin `woo-test-helper.php` once this PR had been merged.